sass/_content.scss

```diff
@@ -5,287 +5,319 @@
-            }
-        }
+    p {
+      margin: 1.5em auto 0;
 
-        .tiled-gallery {
-            margin-top: 30px;
-        }
+      &:first-child {
+        margin-top: 0;
+      }
     }
 
-    &-content {
-
-        > h1,
-        > h2,
-        > h3,
-        > h4,
-        > h5,
-        > h6,
-        > div,
-        > span,
-        > p,
-        > ul,
-        > ol,
-        > blockquote,
-        > hr,
-        > form,
-        > iframe,
-        > address,
-        > table,
-        > dl {
-            @include site-width;
-        }
+    .tiled-gallery {
+      margin-top: 30px;
+    }
+  }
+
+  &-content {
+    > h1,
+    > h2,
+    > h3,
+    > h4,
+    > h5,
+    > h6,
+    > div,
+    > span,
+    > p,
+    > ul,
+    > ol,
+    > blockquote,
+    > hr,
+    > form,
+    > iframe,
+    > address,
+    > table,
+    > dl {
+      @include site-width;
+    }
 
-        > ul,
-        > ol {
-            li {
-                margin-left: 25px;
-            }
-
-            &.inline {
-                li {
-                    display: inline-block;
-                    margin-left: 0;
-                    margin-right: 10px;
-
-                    &:after {
-                        color: #ccc;
-                        content: '\2022';
-                        display: inline-block;
-                        padding-left: 10px;
-                    }
-
-                    &:last-child:after {
-                        display: none;
-                    }
-                }
-            }
+    > ul,
+    > ol {
+      li {
+        margin-left: 25px;
+      }
+
+      &.inline {
+        li {
+          display: inline-block;
+          margin-left: 0;
+          margin-right: 10px;
+
+          &:after {
+            color: #ccc;
+            content: '\2022';
+            display: inline-block;
+            padding-left: 10px;
+          }
+
+          &:last-child:after {
+            display: none;
+          }
         }
+      }
+    }
 
-        > section > h2,
-		> section > h3,
-        > section > p {
-            @include site-width;
-        }
+    > section > h2,
+    > section > h3,
+    > section > p {
+      @include site-width;
+    }
 
-        > p:first-child,
-		> p:first-of-type,
-		> p:first-child:empty + p,
-		> p:first-of-type:empty + p,
-        p.lead {
-			@include rem(font-size, 22px);
+    > p:first-child,
+    > p:first-of-type,
+    > p:first-child:empty + p,
+    > p:first-of-type:empty + p,
+    p.lead {
+      @include rem(font-size, 22px);
+
+      a:hover {
+        border-width: 2px;
+      }
+    }
+
+    > hr {
+      max-width: 250px;
+    }
+
+    > blockquote {
+
+      &.large {
+        @include rem(font-size, 30px);
+        border: 0;
+        max-width: $width-wide;
+        padding: 0;
+        position: relative;
+        text-align: center;
+      }
 
-            a:hover {
-				border-width: 2px;
-			}
-		}
+      &.pullquote {
+        border: 0;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets